rascal2/dialogs/error_dialog.py

```diff
@@ -0,0 +1,21 @@
+"""Dialog for caught Python exceptions."""
+
+from PyQt6 import QtWidgets
+
+
+class ErrorDialog(QtWidgets.QDialog):
```

Comment:
we no longer need a popup error box since the validation errors are now being shown below the boxes in the form, so this won't be relevant anymore - will remove